Engine/EditorUI/Panel/InspectorSubpanel/PanelMaterial.cpp

```diff
@@ -40,7 +45,7 @@ void PanelMaterial::Render(Material* material)
 					{
 						ImGui::SetItemDefaultFocus();
 					}
-					SaveMaterial(material);
+					//SaveMaterial(material);
```

Comment:
Why is this commented now?

Reply:
If this function no longer exist why keep this?